build/task/build-test.js

```diff
@@ -2,11 +2,16 @@ var galv = require('galvatron');
 var gulp = require('gulp');
 var gulpBabel = require('gulp-babel');
 var gulpConcat = require('gulp-concat');
+var gulpFilter = require('gulp-filter');
 
 module.exports = function () {
+  var filterEverythingExceptWebcomponents = gulpFilter(['**/*','!**/webcomponents.js/**/*'], { restore: true });
```

Comment:
workaround for https://babeljs.io/docs/faq/#why-is-this-being-remapped-to-undefined-